site/content/extend/plugins/community-plugin-marketplace.md

```diff
@@ -0,0 +1,90 @@
+    - [PM](https://mattermost.atlassian.net/browse/MM-22224)
+    - [Dev](https://mattermost.atlassian.net/browse/MM-22221)
+    - [Security](https://mattermost.atlassian.net/browse/MM-22225)
+    - [QA](https://mattermost.atlassian.net/browse/MM-22223)
+3. Once all items are checked and reviews made, the community repository is forked under the Mattermost GitHub organization as a private repository. This allows using the existing build tools for releasing new plugin versions. The fork is maintained by the owner.
+4. `/mb cutPlugin --repo $REP --tag $TAG` is run to build, signs and uploads the approved commit of the plugin.
+5. The owner opens a PR, which adds the plugin to `plugins.json` using `generator add $REP $TAG --community`. Only a functional review by one dev and one QA member is needed for this PR.
+6. After the PR is merged, the plugin gets promoted across Mattermost social media and swag sent to the maintainer. If there are multiple maintainers, everyone gets swag.
+
+### Process for updating community plugin to the Marketplace
+The process is the same as for adding a community plugin besides a couple of differences:
+- The following JIRA ticket is used instead
+    - [PM](https://mattermost.atlassian.net/browse/MM-22228)
+    - [Dev](https://mattermost.atlassian.net/browse/MM-22222)
+    - [Security](https://mattermost.atlassian.net/browse/MM-22226)
+    - [QA](https://mattermost.atlassian.net/browse/MM-22227)
+- Swag is not sent again
+- Promotion via social media happens only on outstanding updates.
+- The master branch of the fork synced with the upstream community repository by running:
+
+```sh
+git fetch upstream
+git checkout master
+git --ff-only upstream/master
+```
+
+### Beta plugins
+If a community plugin doesn’t not make it through the review process, it may still be added to the Marketplace and market as “Beta” via a label. The reviewers make a case by case decision if the quality of a plugin is sophisticated enough to be added to the Marketplace. Security and functional review and Item 1, 2, 3 and 5 from the [Product Requirements Checklist](#requirements-for-adding-community-plugin-to-the-marketplace) must be fulfilled for beta plugin.
+
+Given that it must be made clear in the Marketplace UI that a plugin is in beta, all beta plugins must only be visible on Mattermost Servers that support labels, which is Mattermost v5.20.
+
+### Take down Policy
+If a S2 and above security issue or bug that prevents the usage of the plugin for many users is not fixed within 14 days, the plugin gets taken from the Marketplace. It may be resubmitted, once the issue is resolved. Mattermost reserved the right to take down plugins at any time if a fix for a security issue is not forthcoming or the issue is critical enough to justify an immediate takedown.
```

Comment:
```suggestion
If an S2 and above security issue or bug that prevents the usage of the plugin for many users is not fixed within 14 days, the plugin gets taken from the Marketplace. It may be resubmitted, once the issue is resolved. Mattermost reserves the right to take down plugins at any time if a fix for a security issue is not forthcoming or the issue is critical enough to justify an immediate takedown.
```